programmers/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,8 @@
+function solution(s) {
+    
+    const even = s.length / 2;
+    const odd = (s.length / 2) - 0.5;
```

Comment:
오 되게 잘 푸셨네요
```suggestion
    const odd = Math.floor(s.length / 2);
```
Math.floor는 소수점을 버리는 메서드인데요 이렇게도 풀 수 있습니다😁

Reply:
메서드 공부를 열심히 해보겠습니다 감사합니다 !!